src/contexts/KeyringContextProvider.tsx

```diff
@@ -42,14 +42,13 @@ export function KeyringContextProvider(props: KeyringContextProviderProps): Reac
   const [extensionExists, setExtensionExists] = useState<boolean>(false);
   const [accountExists, setAccountExists] = useState<boolean>(false);
 
-  const isDevelopment = Boolean(process.env.REACT_APP_KEYRING_DEV_LOAD_ACCOUNTS);
-
+  const isDevelopment = process.env.REACT_APP_KEYRING_DEV_LOAD_ACCOUNTS === 'true';
```

Comment:
Same here, I'd prefer a separate flag for that.

Reply:
@tomusdrw could you please elaborate about what a `separate flag for that` means?

Reply:
i think i get it ... can we use NODE_ENV in .env file?

Reply:
No, `NODE_ENV` is not the right thing to use here, we need a separate env variable that is specific to either E2E tests or just disabling extension requirement.